<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Auth</class>
 <widget class="QWidget" name="Auth">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>559</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>702</width>
    <height>559</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>702</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/icon/stack.png</normaloff>:/icon/stack.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
border-color: red;
border-style:solid;
border-width:2px;
border-radius:8px;
}
QLineEdit:hover{
background-color:rgb(251, 255, 201)
}

QPushButton:hover{
	background-color:red;
}</string>
  </property>
  <widget class="QLabel" name="lblUSer">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPAss">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-color: rgb(0, 132, 255);
	border-style:solid;
	border-radius:10px;
	border-width:2px;
	color:#000;
	font-size:14px;
}
QPushButton:hover{
	background-color: rgb(11, 166, 9);	
	color:#fff;
}</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="autoRepeatDelay">
    <number>500</number>
   </property>
   <property name="autoRepeatInterval">
    <number>500</number>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-color: rgb(0, 132, 255);
	border-style:solid;
	border-radius:10px;
	border-width:2px;
	font-size:14px;
	color:#000;
}
QPushButton:hover{
	color:#fff;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoRepeatDelay">
    <number>500</number>
   </property>
   <property name="autoRepeatInterval">
    <number>500</number>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtUser">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtPass">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>40</y>
     <width>731</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/img.jpg);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>171</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/lock/lock.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="errorUser" native="true">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:red;
border-style:solid;
border-width:2px;
border-radius:8px;</string>
   </property>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0px;
image: url(:/icon/attantion.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>5</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0px;</string>
    </property>
    <property name="text">
     <string>Invalid UserName  !!!</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="errorPass" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>380</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color:red;
border-style:solid;
border-width:2px;
border-radius:8px;</string>
   </property>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0px;
image: url(:/icon/attantion.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>5</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0px;</string>
    </property>
    <property name="text">
     <string>Invalid PassWord !!!</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>701</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color:red;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>&gt;Login Form</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Auth/user.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Auth/key2.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_10" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.342105 rgba(0, 0, 0, 255), stop:0.952632 rgba(255, 28, 28, 255));</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color:#fff;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>Welcom To Simulation </string>
    </property>
   </widget>
  </widget>
  <zorder>edtPass</zorder>
  <zorder>edtUser</zorder>
  <zorder>lblUSer</zorder>
  <zorder>lblPAss</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget</zorder>
  <zorder>errorUser</zorder>
  <zorder>errorPass</zorder>
  <zorder>widget_7</zorder>
  <zorder>widget_8</zorder>
  <zorder>widget_9</zorder>
  <zorder>widget_10</zorder>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
